app/src/main/java/com/hedvig/app/feature/birthday/data/BirthdayRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.hedvig.app.feature.birthday.data
+
+import io.reactivex.Observable
+
+class BirthdayRepository {
+
+
+    fun fetchBirthDate(): Observable<String> {
+        return Observable.just("199701171234")
```

Comment:
This probably needs to be changed later on, but I'm sure you know that already 😉